src/main/java/com/cs544/project/ProjectApplication.java

```diff
@@ -6,9 +6,18 @@
 import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+import org.springframework.jms.annotation.EnableJms;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
 
 @SpringBootApplication
-//@EnableJpaRepositories(basePackages = "com.cs544.project.repository")
+@EnableJpaRepositories(value = "com.cs544.project.repository")
+@EntityScan(value = "com.cs544.project.domain")
+@ComponentScan(basePackages = {"com.cs544.project"})
```

Comment:
- we don't need an `@ComponentScan` unless we have beans in a package that's not in the same package as the config class.
- same for `@EntityScan` and `@EnableJpaRepositories`

since we don't need them yet, please remove them and we can adjust in future if needed

Reply:
ok